components/feature/webchannel/README.md

```diff
@@ -0,0 +1,32 @@
+# [Android Components](../../../README.md) > Feature > Firefox Accounts WebChannel
+
+ A component providing Firefox Accounts WebChannel functionality as a WebExtension.
+
+## Usage
+
+### Setting up the dependency
+
+Use Gradle to download the library from [maven.mozilla.org](https://maven.mozilla.org/) ([Setup repository](../../../README.md#maven-repository)):
+
+```Groovy
+implementation "org.mozilla.components:feature-webchannel:{latest-version}"
+```
+
+### Integration
+
+Initializing the feature:
+
+```kotlin
+val WebChannelViewFeature = WebChannelViewFeature(
+    context,
+    engine,
+    sessionManager
+)
+
+```
+
```

Comment:
add usage

Reply:
> rename this to fxa-webchannel

^ also in `implementation "org.mozilla.components:fxa-webchannel:{latest-version}"`